source/admin_manual/list_of_events.rst

```diff
@@ -1435,354 +869,222 @@ HTTP Server
-| session             | Session ID of the IMAP connection                    |
-+---------------------+------------------------------------------------------+
-| mailbox             | Mailbox name where hibernation was started in.       |
-+---------------------+------------------------------------------------------+
-| local_ip            | IMAP connection's local (server) IP                  |
-+---------------------+------------------------------------------------------+
-| local_port          | IMAP connection's local (server) port                |
-+---------------------+------------------------------------------------------+
-| remote_ip           | IMAP connection's remote (client) IP                 |
-+---------------------+------------------------------------------------------+
-| remote_port         | IMAP connection's remote (client) port               |
-+---------------------+------------------------------------------------------+
-
-.. _imap_hibernate_client_unhibernated:
-
-imap_client_unhibernated
-------------------------
-
-Event emitted when an IMAP client is unhibernated or when the unhibernation attempt failed.
-Note that for failures this event can be logged by either imap or imap-hibernate process depending on which side the error was detected in.
-
-See also imap process's :ref:`imap_client_unhibernated` event.
-
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| hibernation_usecs   | Number of microseconds how long the client was       |
-|                     | hibernated.                                          |
-+---------------------+------------------------------------------------------+
-| error               | Reason why unhibernation failed.                     |
-+---------------------+------------------------------------------------------+
-
-imap_client_unhibernate_retried
--------------------------------
-
-Event emitted when an IMAP client is attempted to be unhibernated, but imap processes are busy and the unhibernation attempt is retried.
-This event is sent each time when retrying is done.
-The :ref:`imap_client_unhibernated` event is always still sent when unhibernation either succeeds or fails permanently.
-
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| error               | Reason why unhibernation attempt failed.             |
-+---------------------+------------------------------------------------------+
 
 Mail Delivery
 =============
 
 Events emitted on mail delivery.
 
-Common fields:
+.. dovecot_event:field_group:: mail_delivery
+   :inherit: smtp_recipient
```

Comment:
not true if emitted by dovecot-lda..

Reply:
See above.  (I don't see how this kind of event is useful unless the assumption is made that the event receiving client is able to accept different values for the same event.  In other words, a single event ID can have multiple payload signatures.  At a minimum, there needs to be a decision on how to document this - This would require some kind of dummy event link that doesn't currently exist...)